.github/workflows/test-inputs.yml

```
@@ -35,6 +35,17 @@ jobs:
         run: |
           echo "Error! The action should have failed because of checking the dummy crate, but it has not!"
           exit 1
+      - name: Run the action on ref_slice fork and one dummy crate (allowed to fail)
```

```suggestion
      - name: Run the action on ref_slice fork and one dummy crate (expected to fail)
```
"allowed" to me sounds like "it might fail or it might not" and here it seems like we want something a bit stronger

---

Good idea! I've changed all occurences